.github/workflows/test.yml

```diff
@@ -29,7 +27,8 @@ jobs:
       - name: Run unit tests with code coverage and Thread Sanitizer
         run: swift test --enable-test-discovery --filter=^PostgresNIOTests --sanitize=thread --enable-code-coverage
       - name: Submit coverage report to Codecov.io
-        uses: vapor/swift-codecov-action@v0.2
+        if: "!contains(matrix.container, 'nightly')"
+        uses: vapor/swift-codecov-action@v0.2.1
```

Comment:
The `v0.2` tag points to the latest "0.2.x" version, not `0.2.0`; it has the effect of SPM's `upToNextMinor()`, very much on purpose, please keep it that way 🙂

Reply:
Good to know!